fhir-bucket/src/main/java/com/ibm/fhir/bucket/scanner/ResourceHandler.java

```diff
@@ -152,227 +124,9 @@ public boolean process(ResourceEntry entry) {
-            String id = parts[7];
-            result = new ResourceIdValue(resourceType, id);
-        } else if (parts.length == 4) {
-            String resourceType = parts[0];
-            String id = parts[1];
-            result = new ResourceIdValue(resourceType, id);
-        } else {
-            result = null;
-        }
-        
-        return result;
-    }
-    
-    private boolean processLocation(ResourceEntry re, String location, int responseTimeMs) {
-        boolean result = false;
-        // the response was empty, so in this case we need to extract the id from
-        // the location header, which means cracking the string into parts:
-        // https://localhost:9443/fhir-server/api/v4/DiagnosticReport/173eed87a99-605de23b-266d-4b4d-b64f-31e769fda112/_history/1
-        String[] parts = location.split("/");
-        if (parts.length == 10) {
-            String resourceType = parts[6];
-            String id = parts[7];
-            logger.info("[" +re.toString() + "] new " + resourceType + "/" + id + " [took " + responseTimeMs + " ms]");
-            dataAccess.recordLogicalId(resourceType, id, re.getJob().getResourceBundleLoadId(), re.getLineNumber(), responseTimeMs);
-            result = true;
-        }
-        
-        return result;
-    }
-
-    /**
-     * Record the error in the database
-     * @param re
-     * @param response
-     */
-    protected void processBadRequest(ResourceEntry re, FhirServerResponse response) {
-        
-        if (logger.isLoggable(Level.FINE)) {
-            // dump the resource and full operational outcome to the log
-            logger.fine(re.getJob().getObjectKey() + "[" + re.getLineNumber() + "]: "
-                + resourceToString(re.getResource()));
-            logger.fine(re.getJob().getObjectKey() + "[" + re.getLineNumber() + "]: "
-                + response.getOperationalOutcomeText());
-        }
-        
-        List<ResourceBundleError> errors = new ArrayList<>();
-        errors.add(new ResourceBundleError(re.getLineNumber(), response.getOperationalOutcomeText(), 
-            response.getResponseTime(), response.getStatusCode(), response.getStatusMessage()));
-        
-        dataAccess.recordErrors(re.getJob().getResourceBundleLoadId(), re.getLineNumber(), errors);
-    }
-
-    /**
-     * Render the resource as a string (for logging)
-     * @param resource
-     * @return
-     */
-    private String resourceToString(Resource resource) {
-        ByteArrayOutputStream os = new ByteArrayOutputStream(4096);
-        try {
-            FHIRGenerator.generator(Format.JSON, false).generate(resource, os);
-            return new String(os.toByteArray(), StandardCharsets.UTF_8);
-        } catch (FHIRGeneratorException e) {
-            throw new IllegalStateException(e);
-        }
-    }
 
     /**
-     * 
+     * Perform any initialization after things have been wired up
      */
     public void init() {
     }
```

Comment:
Where is this called, and is this more than dead code?

Reply:
It was called from Main, but I removed it.